api/URLUtilsReadOnly.json

```diff
@@ -514,6 +523,7 @@
       "toString": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/URLUtilsReadOnly/toString",
+          "spec_url": "https://url.spec.whatwg.org/#URL-stringification-behavior",
```

Comment:
This didn't seem quite right. This mixin was actually removed in https://github.com/whatwg/html/pull/187 so should we unmix it in BCD and just not add the spec URLs?

Reply:
> This didn't seem quite right. This mixin was actually removed in [whatwg/html#187](https://github.com/whatwg/html/pull/187) so should we unmix it in BCD and just not add the spec URLs?

You’re right. So I’ve opened https://github.com/mdn/browser-compat-data/pull/10347 for dropping `api/URLUtilsReadOnly.json` from BCD, and moving its members to `api/WorkerLocation.json`. (I’ve also opened https://github.com/mdn/content/pull/4772 for the corresponding MDN changes.)

And we don’t need to block this PR on that, so I’ve pushed an amended commit which drops the `api/URLUtilsReadOnly.json` spec-URL additions from this PR.

Reply:
Thanks Mike!

Reply:
Indeed, thanks Mike!